skiko/src/commonMain/kotlin/org/jetbrains/skia/paragraph/TextStyle.kt

```diff
@@ -154,10 +154,12 @@ class TextStyle internal constructor(ptr: NativePointer) : Managed(ptr, _Finaliz
         return this
     }
 
-    val shadows: Array<org.jetbrains.skia.paragraph.Shadow>
+    val shadows: Array<Shadow>
         get() = try {
             Stats.onNativeCall()
-            _nGetShadows(_ptr)
+            Shadow.fromInteropPointer(_nGetShadowsCount(_ptr) * 5) {
```

Comment:
I'd rather move multiplication by 5 into `Shadow.fromInteropPointer` body to avoid leaking internals

Reply:
Totally makes sense, pushed